optax/_src/alias.py

```diff
@@ -56,32 +57,85 @@ def adagrad(
   )
 
 
-def adam(learning_rate: ScalarOrSchedule,
-         b1: float = 0.9,
-         b2: float = 0.999,
-         eps: float = 1e-8,
-         eps_root: float = 0.0) -> GradientTransformation:
+def adam(
+    learning_rate: ScalarOrSchedule,
+    b1: float = 0.9,
+    b2: float = 0.999,
+    eps: float = 1e-8,
+    eps_root: float = 0.0) -> GradientTransformation:
+  """The classic Adam optimiser.
+
+  Adam is an SGD variant with learning rate adaptation. The `learning_rate`
+  used for each weight is computed from estimates of first and second order
+  moments of the gradients (using suitable exponential moving averages).
+
+  References:
+    [Kingma et al, 2014](https://arxiv.org/abs/1412.6980)
+
+  Args:
+    learning_rate: this is a fixed global scaling factor.
+    b1: the decay used to track the first moment of past gradients.
+    b2: the decay used to track the second moment of past gradients.
+    eps: a small constant applied to denominator outside of the square root
+      (as in the Adam paper) to avoid dividing by zero when rescaling.
```

Comment:
Thanks for improving the optimizer docs @mtthss 👋  . Do you think adding default values here would help some users?

```
    b1: float = 0.9,
    b2: float = 0.999,
    eps: float = 1e-8
```

For example:

```suggestion
    b1: the decay used to track the first moment of past gradients (default: 0.9).
    b2: the decay used to track the second moment of past gradients (default: 0.999).
    eps: a small constant applied to denominator outside of the square root
      (as in the Adam paper) to avoid dividing by zero when rescaling (default: 1e-8).
```

Similar to:
- https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/optimizer_v2/adam.py
- https://github.com/tensorflow/swift-apis/blob/main/Sources/TensorFlow/Optimizers/MomentumBased.swift
- https://github.com/pytorch/pytorch/blob/master/torch/optim/adam.py